daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -299,6 +299,40 @@ auth:
   secretStore: <SECRET_STORE_NAME>
 ```
 
+## Consuming from multiple topics
```

Comment:
Is it best to call this title "Optimizing when consuming multiple topics" or similar?

Reply:
Initially I had put this down as "Optimizing" but based on the feedback on the open issue around this behaviour as well as the conversations I've had with other developers, this feels/is like a workaround and not an optimisation. Because of that I opted not to put anything other a simple title, as listed above.

Thoughts?

Reply:
Sounds good. Then let's leave it as is.